gradle/libs.versions.toml

```diff
@@ -1,14 +1,15 @@
 [versions]
-kotlin = "1.9.24"
-kotlin-wrappers = "1.0.0-pre.748"
-kotlinx-io = "0.3.4"
-kotlinx-benchmark = "0.4.10"
-kotlinx-resources = "0.4.2"
+kotlin = "2.1.20"
+kotlin-wrappers = "2025.5.2"
+kotlinx-io = "0.7.0"
+kotlinx-benchmark = "0.4.13"
+kotlinx-resources = "0.10.0"
 antlr4 = "4.13.1"
 dokka = "1.9.20"
-researchgate-release = "3.0.2"
-maven-publish = "0.28.0"
-gradle-plugin-publish = "1.2.1"
+researchgate-release = "3.1.0"
+maven-publish = "0.30.0"
+gradle-plugin-publish = "1.3.0"
+antlr-kotlin = "1.0.2"
```

Comment:
Should antlr-kotlin comes from version in gradle.properties somehow?

Reply:
No no, this is correct, we want to pull that version (the last properly working version) from a remote repo.